scripts/ldmx-recompileAndFire.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+############################# Usage #############################
+# ldmx ./scripts/ldmx-recompileAndFire.sh
+# for changing the defaults
+# ldmx setenv LDMX_COMPILE_CORES=<num cores to be used>, i.e.
+# ldmx setenv LDMX_COMPILE_CORES=16
+# or
+# ldmx setenv LDMX_COMPILE_BUILD=<build location>
+
+FIREINPUT=$1
+
+if [ -z "$LDMX_COMPILE_CORES" ]; then
+    LDMX_COMPILE_CORES=$(nproc)
+fi
+
+if [ -z "$LDMX_COMPILE_BUILD" ]; then
+    LDMX_COMPILE_BUILD=${LDMX_BASE}/ldmx-sw/
+fi
+
+echo "-- Compiling ldmx-sw in ${LDMX_COMPILE_BUILD} with ${LDMX_COMPILE_CORES} cores, and running "
+
+# Compile ldmx-sw
+cmake -B ${LDMX_COMPILE_BUILD}/build -S ${LDMX_COMPILE_BUILD}
+cmake --build ${LDMX_COMPILE_BUILD}/build --target install -j=${LDMX_COMPILE_CORES}
+# Run fire on the input config
+fire $FIREINPUT
```

Comment:
Instead of storing only the first argument, we should pass the entire command line to fire. For this reason, I'm just giving `$@` to `fire` directly (no double quotes because we want `fire` to see the split command line and not just one string).

```suggestion
fire $@
```